requirements.txt

```diff
@@ -1 +1,2 @@
 pyyaml
+prometheus_api_client
```

Comment:
We'll need to add the version number to be greater than the one ( once the pr gets merged and a new release comes out )